### DISCUSSION TITLE
CustomTkinter in BeeWare -   if Version(darkdetect.__version__) < Version("0.3.1"):

### DISCUSSION THREAD

**Discussion by Twins285** [TomSchimansky/CustomTkinter#1561] [Q&A] [answered]
Good times!  Sorry in advance for my Dagestani accent in English spelling).

I am studying programming on my own and only for the first couple of months, so I can't figure out this situation in any way:
I built an application in the beautiful CustomTkinter and everything in it works great. I decided to try to make it for Android (that would be very cool) with BeeWare, but I faced an impossible task -

/beeware-tutorial/beeware-venv/lib/python3.8/site-packages/customtkinter/windows/widgets/appearance_mode/appearance_mode_tracker.py:9: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  if Version(darkdetect.__version__) < Version("0.3.1"):

darkdetect is 0.8.0.
After reading all sorts of discussions on similar issues, I couldn't come up with anything meaningful.
How to live with this? ))) Is it even possible to solve it?
 Thanks for your help))

**Comment by AlbertSolomon** [comment]
It appears that the version of ``darkdetect`` you are using may be outdated and needs to be updated, to verify this run ``pip show darkdetect``. Additionally, there may be issues with the ``darkdetect`` library used in customtkinter as it is not supported on Android. For more information about darkdetect, click  👉🏽 [darkdetect 0.8.0 ](https://pypi.org/project/darkdetect/).

**Comment by TomSchimansky** [comment, marked as answer]
You can always just download the customtkinter folder from GitHub, put it next to your main file and use it locally. Then you could just go into the appearance_mode_tracker.py file and delete the version check, I think I catch any errors with darkdetect, it should work fine anyways. (besides the "system" mode for the appearance mode)

**Reply by Twins285** [reply]
The issue was resolved by removing the darkdetect version check, but importing it remains necessary. The difficulties arise in BeeWare itself. As soon as I figure it out, I'll post what I've got). Thanks for the support!
